### PR TITLE
remove debug label from documentation

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR removes the *debug_label*-option from the documentation.

### CODE REVIEW

**Review by terrymyc — APPROVED** [review]

File: docs/source/intro/datasets.rst [hunk 1]
```diff
@@ -96,13 +95,16 @@ not have access you might want to read this `blog post
...
-names are allowed, see the table. The use is twofold:
+names are allowed, see the table. It can be used for:
 
 - **Screening**: In ASReview LAB, if labels are available for a part of the
   dataset (see :doc:`partly labeled data <../features/pre_screening>`), the
   labels will be automatically detected and used for prior knowledge. The first
   iteration of the model will then be based on these decisions and used to
   predict relevance scores for the unlabeled part of the data.
+- **Exploration**: You can explore an existing fully labeled dataset in the Exploraton
+  Mode. The relevant records will be displayed in green during screening.
```
Comment (J535D165) [inline docs/source/intro/datasets.rst:106]: @terrymyc can you rephrase/fix this?

Comment (terrymyc) [inline docs/source/intro/datasets.rst:106]:
```suggestion
- **Exploration**: You can explore a completely labeled dataset in the Exploration
  Mode. The relevant/irrelevant label in the dataset will be displayed on each record.
```

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks, nice deletions.
